src/plugins/mer/merplugin.cpp

```diff
@@ -154,6 +154,10 @@ bool MerPlugin::initialize(const QStringList &arguments, QString *errorString)
     RunControl::registerWorker<RemoteLinuxQmlProfilerSupport>(QML_PROFILER_RUN_MODE, constraint);
     //RunControl::registerWorker<RemoteLinuxPerfSupport>(PERFPROFILER_RUN_MODE, constraint);
 
+    Utils::globalMacroExpander()->registerVariable("SailfishOSVariant",
+                                                   tr("Category of a wizard for %1 files").arg(Sdk::osVariant()),
+                                                   []() { return Sdk::osVariant(); });
```

Comment:
I would immediately register all the variables os/ide/sdk variant (similarly to how it is done under sfdk)